go.mod

```diff
@@ -1,7 +1,18 @@
 module github.com/blocky/basm-go-sdk
 
-go 1.22.6
+go 1.23.8
 
-require github.com/mailru/easyjson v0.9.0
+require (
+	github.com/mailru/easyjson v0.9.0
+	github.com/rogpeppe/go-internal v1.14.1
+	github.com/stretchr/testify v1.10.0
```

Comment:
adding testify added a lot of dependencies.  Maybe we avoid on this on?

Reply:
addressed!